zcash_client_backend/src/proto.rs

```diff
@@ -1,5 +1,3 @@
-//! Generated code for handling light client protobuf structs.
```

Comment:
If you initially generate code, and then stop generating it and edit it to make changes that couldn't be generated, then it is no longer generated.

The comment misled me in two ways:
* I thought it was still generated, like several other files similar to it, and therefore free of manual changes.
* I thought that I couldn't change it, lest the changes be overwritten.

Reply:
We don't stop generating the code. Everything in `zcash_client_backend/src/proto/` is fully generated. `zcash_client_backend/src/proto.rs` itself is _not_ generated and never has been, but originally it was simply a module that defined the generated ones as sub-modules.

Reply:
It's a misleading comment for the top-level module, since the module contains both generated code and hand-written adapters from the generated types to the domain types. I don't have a strong opinion about how to fix it though.